cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyDetails.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2019 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.detector;
+
+import static com.linkedin.kafka.cruisecontrol.KafkaCruiseControlUtils.utcDateFor;
+import static com.linkedin.kafka.cruisecontrol.detector.notifier.KafkaAnomalyType.GOAL_VIOLATION;
+import com.linkedin.cruisecontrol.detector.AnomalyType;
+import com.linkedin.kafka.cruisecontrol.detector.notifier.KafkaAnomalyType;
+import com.linkedin.kafka.cruisecontrol.servlet.response.JsonResponseField;
+import com.linkedin.kafka.cruisecontrol.servlet.response.JsonResponseClass;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@JsonResponseClass
+public class AnomalyDetails {
+@JsonResponseField
+protected static final String STATUS_UPDATE_MS = "statusUpdateMs";
+@JsonResponseField(required = false)
```

Comment:
`STATUS_UPDATE_DATE ` and `DETECTION_DATE ` should not be annotated since they are only used in plaintext response.